resources/services/elasticbeanstalk/environments.go

```diff
@@ -450,6 +458,17 @@ func fetchElasticbeanstalkEnvironmentLinks(ctx context.Context, meta schema.Clie
 	return nil
 }
 
+// It takes a few minutes for an environment to be terminated
+// This ensures we don't error while trying to fetch related resources for a terminated environment
+func IsInvalidParameterValueError(err error) bool {
```

Comment:
Fix for https://github.com/cloudquery/cq-provider-aws/issues/621#issuecomment-1103846841